Fix: GitLab backup script should start instance after completing (#4819) (#4820)
DataBiosphere/azul: OPERATOR.rst

	git checkout -b gitlab/yyyy-mm-dd/<GitLab version> github/develop
	_select dev.gitlab

> Reference the :ref:`_volumes-backup` section in this manual to use the

Reviewer: Link doesn't work for me.

have been successfully updated, file a PR with the changes against the
``develop`` branch and request review from the lead.

Backup GitLab volumes
^^^^^^^^^^^^^^^^^^^^^

> Use the following script to create a snapshot of the storage volume attached to
> the GitLab instance. The script will stop the instance, create a properly tagged
> snapshot of the GitLab EBS volume and restart the instance. Use the
> `--no-restart` flag if this is a GitLab and ClamAV update, to leave the instance
> stopped after the snapshot has been created. Otherwise, run::

Reviewer: ```suggestion
Use the `create_gitlab_snapshot` script to back up the EBS data volume attached to
each of our GitLab instances. The script will stop the instance, create a snapshot of the GitLab EBS volume, tag the snapshot and restart the instance::
```

	python scripts/create_gitlab_snapshot.py

> 

Reviewer: ```suggestion

For GitLab or ClamAV updates, use the
`--no-restart` flag in order to leave the instance
stopped after the snapshot has been created. There is no point in starting the instance only to have the  update terminate it.
```

have been successfully updated, file a PR with the changes against the
``develop`` branch and request review from the lead.

Backup GitLab volumes
^^^^^^^^^^^^^^^^^^^^^

> Use the `create_gitlab_snapshot` script to back up the EBS data volume attached

Reviewer: ```suggestion
Use the ``create_gitlab_snapshot.py`` script to back up the EBS data volume attached
```

to each of our GitLab instances. The script will stop the instance, create a

> snapshot of the GitLab EBS volume, tag the snapshot and restart the instance::

Reviewer: ```suggestion
snapshot of the GitLab EBS volume, tag the snapshot and finally restart the instance::
```

	python scripts/create_gitlab_snapshot.py

> For GitLab or ClamAV updates, use the `--no-restart` flag in order to leave the

Reviewer: ```suggestion
For GitLab or ClamAV updates, use the ``--no-restart`` flag in order to leave the
```

instance stopped after the snapshot has been created. There is no point in

> starting the instance only to have the  update terminate it.

Reviewer: ```suggestion
starting the instance only to have the update terminate it.
```

Reviewer: ```suggestion
starting the instance only to have the  update terminate it again.
```
